setup.cfg

```diff
@@ -12,6 +12,10 @@ lint-reports = no
 lint-disable = locally-disabled,star-args
```

Comment:
Does someone know what these configurations are about? They do not configure `pylint` from what I can tell.

Reply:
Yes, I'll open a PR in a moment.